### PR TITLE
Ospp 2024/feat graphic dialogue

### PR DESCRIPTION
**Pull request by Fleurxxx**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **New Features**
  - Introduced image upload capabilities within the chat interface.
  - Enhanced AI model selection logic for improved clarity and control.

- **Bug Fixes**
  - Updated validation logic and terminology for token handling.

- **Documentation**
  - Updated component communication to include new events and properties.

- **Chores**
  - Deprecated previous AI model options and introduced a new Kimi model in the settings.

### CODE REVIEW

File: packages/plugins/robot/src/TokenDialog.vue [hunk 1]
```diff
@@ -15,34 +15,37 @@ export default {
     TinyFormItem: FormItem
   },
   props: {
-    dialogVisible: Boolean
+    dialogVisible: Boolean,
+    currentModel: Object
```
[the remark this answers is not in the document]
Comment (CatsAndMice) [reply packages/plugins/robot/src/TokenDialog.vue]: @Fleurxxx 记得修改

File: packages/plugins/robot/src/Main.vue [hunk 2]
```diff
@@ -231,16 +251,17 @@ export default {
           sessionProcess.displayMessages.push(respDisplayMessage)
           messages.value[messages.value.length - 1].content = originalResponse.choices?.[0]?.message.content
           setContextSession()
-          if (schema?.schema) {
-            createNewPage(schema.schema)
-          }
+          // TODO：返回schema格式的代码
```
Comment (CatsAndMice) [inline packages/plugins/robot/src/Main.vue:254]: schema还不能生成吗？

File: packages/plugins/robot/src/Main.vue [hunk 3]
```diff
@@ -372,20 +480,31 @@ export default {
...
 
     const changeModel = (model) => {
-      if (selectedModel.value.value !== model.value) {
+      if (currentModel.value !== model.value) {
         confirm({
           title: '切换AI大模型',
           message: '切换AI大模型将导致当前会话被清空，重新开启新会话，是否继续？',
           exec() {
             selectedModel.value = model
+            currentModel = model
```
Comment (CatsAndMice) [inline packages/plugins/robot/src/Main.vue:489]: currentModel 是响应式变量，赋值不需要.value吗？

File: packages/plugins/robot/src/TokenDialog.vue [hunk 4]
```diff
@@ -84,17 +89,17 @@ export default {
       <tiny-alert
         :icon="TinyIconAssociation"
         :closable="false"
-        description="当前AI大模型为使用文心一言：ERNIE-Bot-turbo"
+        :description="`当前AI大模型为使用${model.label}`"
```
Comment (CatsAndMice) [inline packages/plugins/robot/src/TokenDialog.vue]: 描述不通顺，当前AI大模型${model.label}，展示大模型名即可。类似ERNIE-Bot-turbo这类的东西可以去掉，不需要让用户看到

File: packages/plugins/robot/src/Main.vue (hunk 2 above)
Comment (Fleurxxx) [reply packages/plugins/robot/src/Main.vue:254]: 在另一个分支上做的这个功能

**Review by CatsAndMice — APPROVED** [review]

**Comment by Fleurxxx** [comment]
@hexqi H可以帮我看一下这个pr吗

**Review by hexqi — APPROVED** [review]

**Pull request merged**